mlrun/api/api/endpoints/feature_sets.py

```diff
@@ -0,0 +1,85 @@
+from http import HTTPStatus
+from typing import List
+
+from fastapi import APIRouter, Depends, Response, Query
+from sqlalchemy.orm import Session
+
+from mlrun.api import schemas
+from mlrun.api.api import deps
+from mlrun.api.utils.singletons.db import get_db
+from .utils import parse_reference
+
+router = APIRouter()
+
+
+@router.post(
+    "/projects/{project}/feature_sets", response_model=schemas.FeatureSetCreateOutput
+)
+def create_feature_set(
+    project: str,
+    feature_set: schemas.FeatureSet,
+    versioned: bool = False,
+    db_session: Session = Depends(deps.get_db_session),
+):
+    feature_set_id = get_db().create_feature_set(
+        db_session, project, feature_set, versioned
+    )
+    return schemas.FeatureSetCreateOutput(
+        uid=feature_set_id, name=feature_set.metadata.name
+    )
+
+
+@router.put("/projects/{project}/feature_sets/{name}/references/{ref}")
```

Comment:
I know this is a little bit nit picking, but again, as a rule of thumb, let's prefer to use full names, no reason to save characters, please fix in the get as well
```suggestion
@router.put("/projects/{project}/feature_sets/{name}/references/{reference}")
```

Reply:
👍